CHANGELOG.md

```diff
@@ -72,6 +72,13 @@ Latest
   animate N-Dimensional data with the associated WCS object.
 * Moved Docs to docs/ to follow the astropy style
 * Added SunPy specific warnings under util.
+* SunPy coordinate frames can now be transformed to and from Astropy coordinate frames
```

Comment:
Definitely not, since HGS is not limited to just the disk!  After all, we specify the position of the Earth in it.